crane/views/v1.py

```diff
@@ -48,6 +48,23 @@ def ping():
     return response
 
 
+@section.route('/repositories')
```

Comment:
Are we going to rely on Apache to restrict access to this view? Or does the view itself need to check for auth?